app/controllers/components/course/users_component.rb

```diff
@@ -0,0 +1,13 @@
+class Course::UsersComponent
+  include Course::ComponentHost::Component
+
+  sidebar do
+    [
+      {
+        title: t('layouts.course_users.title'),
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.